packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -9,7 +8,8 @@ export type Files = File[];
 export interface UploaderButtonProps {
   acceptedFileTypes: string[];
   multiple?: boolean;
-  onClick: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onFileChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  className?: string;
```

Comment:
Related to my comment on the last PR. We should be able to just pass the upload function to the `Button` without the need for the `Button` to handle any sort of logic

Reply:
I'm not sure I understand what you are proposing. The logic isn't going to the button, it's going to the input, and that logic is being passed from the parent.